org.eclipse.scanning.points/src/org/eclipse/scanning/points/PointGeneratorService.java

```diff
@@ -196,25 +199,21 @@ private static void readExtensions(Map<Class<? extends IScanPathModel>, Class<?
 		}
 	}
 
-	private List<IPointContainer> wrap(Collection<?> regions) throws GeneratorException {
-
+	private List<IPointContainer> wrap(Collection<?> regions) {
 		if (regions==null || regions.isEmpty()) return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)